genai-perf/README.md

```diff
@@ -469,14 +469,10 @@ Add a custom header to the requests. Headers must be specified as
 (default: `None`)
 
 ##### `--input-file <path>`
-
-The input file or directory containing the content to use for
-profiling. Each line should be a JSON object with a 'text' or 'image' field
-in JSONL format. Example: {\"text\": \"Your prompt here\"}"
-To use synthetic files for a converter that needs
-multiple files, prefix the path with 'synthetic:', followed by a
-comma-separated list of filenames. The synthetic filenames should not have
-extensions. For example, 'synthetic:queries,passages'.
+The input file or directory for profiling. Each line should be a JSON object with a `text` or `image` field
+in JSONL format. Example: `{"text": "Your prompt here"}`. To use synthetic files, prefix with `synthetic:` followed by
+a comma-separated list of filenames without extensions (Example: `synthetic:queries,passages`).
+To use a payload file with a fixed schedule workload, prefix with `payload:` followed by the filename (Example: `payload:input.jsonl`).
```

Comment:
Again, I'd love for you and @debermudez to chat a bit about this design. My gut says that we should move all this overloading to a tutorial somewhere. Like just mention the base case, but then have tutorials on how to use synthetic and payload for those specific use cases. I appreciate you shortening this, but I think we need it to only have the general case to be simpler for a casual user.

Reply:
This has always felt like a not intuitive way to pass these args. I have never really felt this approach was ergonomic and I think we are just piling on here when we should take a step back and redesign the way we pass these arguments. 
 I think this goes away entirely when we swap to using a configuration file because it is more flexible but I need to review it. I think our time is better spent getting this feature done and revisiting after the configuration file introduction instead of a new document that is not going to be valid in 4-6 weeks. 
That is my personal take but im open to other approaches.

Reply:
Are we no longer going to have arg descriptions with the configuration file approach? And is that going to be the only way of feeding args now?

Reply:
That is a good point.
So the ask is to highlight the various options and link to a tutorial doc rather than having the explanation here?

Reply:
It was. It's not a must, just that I'd rather the special options be highlighted in the associated tutorials (e.g. for synthetic payloads, for timing strategy files, etc.). Otherwise, it'll be confusing for the user.

Having these here is helpful as a developer to remember these options exist, but perhaps we switch that to a comment. Or we ultimately clean it up to be more commonsense for GA, e.g. having two arguments: --input-file-type and --input-file.